modules/bookshelf.js

```diff
@@ -0,0 +1,66 @@
+const storageKey = 'awesomeBooks';
+const timeNow = new Date();
+
+export default class BookShelf {
+  constructor() {
+    this.arrBooks = [];
+  }
+
+  addBook(newTitle, newAuthor) {
+    const newBook = {
+      id: timeNow.getTime(),
+      title: newTitle,
+      author: newAuthor,
+    };
+
+    this.arrBooks.push(newBook);
+  }
+
+  removeBook(id) {
+    this.arrBooks = this.arrBooks.filter((book) => book.id !== id);
+  }
```

Comment:
- [ ] When I try to add or remove a book from your application then it is adding it and removing it :+1: but the problem is that your page refreshes on add or remove which is not expected from a Single Page Application :thinking: and your application is a Single Page Application and it should reload the page on the very first time but after that, on any event, it should not reload the page so kindly fix this issue :+1: